charter.md

```diff
@@ -14,22 +14,33 @@ proxy. Moreover, it would not introduce potentially redundant or unnecessary rec
-multiple proxied stream- and datagram-based flows inside an HTTPS connection. These mechanism(s) are collectively called
+multiple proxied stream- and datagram-based flows inside an HTTP connection. These mechanism(s) are collectively called
 MASQUE. The group will specify HTTP and/or HTTP/3 extensions to enable this functionality.
 
-The group will focus on a limited set of client-initiated services: (1) UDP CONNECT and (2) IP proxying.
-Server-initiated services are out of scope. The working group will first deliver a protocol solution for UDP CONNECT and
-a requirements document for IP proxying. Once both are complete, the working group will focus on a protocol solution for
-IP proxying.
+The group will focus on extensions for a limited set of client-initiated services: (1) UDP CONNECT and (2) IP proxying.
+Server-initiated services are out of scope.
 
-The working group will consider fallback to versions of HTTPS that operate over TCP as a mitigation to UDP or HTTP/3
-blocking. Moreover, the working group will consider implications of tunneling protocols with congestion control and loss
-recovery over MASQUE, and may issue recommendations accordingly. New congestion control and loss recovery algorithms are
-out of scope.
+Exercising the extension points defined by CONNECT-UDP and CONNECT-IP helps to make it easier to support new use cases
+or accommodate changes in the environment in which these protocols are deployed. The initial set of extensions will be
+in support of UDP listening and QUIC-aware proxying. Additional extensions that provide missing functionality, improve
+performance, or otherwise ease deployability for use cases may be adopted where there are multiple implementation
+and/or deployment proponents. 
 
-Multicast support is out of scope. However, the group may specify extension points that would enable future work on
-multicast. Specifying proxy server discovery mechanisms is also out of scope. However, the group may consider features,
-such as proxy server identifiers, that aid future discovery mechanisms.
+Extensions to HTTP Datagrams will be coordinated with HTTPBIS. Extensions that solely relate to generic proxying
+functionality, and are not specific to the core MASQUE documents, are out of scope. 
+
+Specifying proxy server discovery mechanisms is out of scope. New congestion control and loss recovery algorithms are
+also out of scope. However, the working group will consider implications of tunneling protocols with congestion control
+and loss recovery over MASQUE proxies, and may issue recommendations accordingly. 
+
+The working group will consider how the protocols it defines might operate over versions of HTTP that use TCP rather
+than QUIC, for use when QUIC is unavailable. This might include defining alternative extensions specifically for use in
+these HTTP versions.
+
+Multicast is out of scope. Extensions providing generic capabilities that might include multicast are acceptable on the
```

Comment:
Doesn't hurt to be as specific as possible, don't confuse it with "application-layer multicast" etc

```suggestion
IP multicast is out of scope. Extensions providing generic capabilities that might include IP multicast are acceptable on the
```